games-action/heretic2-demo/heretic2-demo-1.06a-r1.ebuild

```diff
@@ -24,7 +24,7 @@ RDEPEND="
 BDEPEND="games-util/loki_patch"
 
 dir=opt/${PN}
-QA_PREBUILT="${dir:1}/*"
```

Comment:
I think we can drop multilib as an inherit now btw

Reply:
thanks, I've dropped it